tests/features/template/default/general.feature

```diff
@@ -0,0 +1,7 @@
+Feature: documentation index
```

Comment:
I think this was not intended to be included?

Reply:
Ah indeed, that can be removed again